docs/source/rule-testing.rst

```diff
@@ -73,43 +73,63 @@ For example, to replace a time based field with ``last_hour``::
     ]
   }
 
+
 Running Tests
-~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~
+
+Various tests can be run via the ``stream_alert_cli.py`` script. These tests include the ability to validate rules for
+accuracy and alert outputs for proper configuration.
+
+When adding new rules, it is only necessary to run tests for the **rule processor**. If making code changes to the alert
+processor, such as adding a new output integration to send alerts to, tests for the **alert processor** should also be performed.
+
+To run integration tests for the **rule processor**::
+
+  $ python stream_alert_cli.py lambda test --processor rule
+
+To run integration tests for the **alert processor**::
+
+  $ python stream_alert_cli.py lambda test --processor rule
+
+To run end-to-end integration tests for **both processors**::
+
+  $ python stream_alert_cli.py lambda test --processor all
+
+Integration tests can be restricted to **specific rules** to reduce time and output::
+
+  $ python stream_alert_cli.py lambda test --processor all --rules <rule_01> <rule_02>
```

Comment:
This is weird because above you say if you want to test rules, you only need to test the rule processor, hence using `--processor rule`. Yet here, when you're testing a subset of rules, you're using `--processor all` - why?